templates/rapidsai-core/partials/os_pkgs.dockerfile.j2

```diff
@@ -1,11 +1,13 @@
 {% if "centos" in os %}
 RUN yum install -y \
-      openssh-clients
+      openssh-clients \
+      openmpi-devel
```

Comment:
```suggestion
      openmpi-devel \
      && yum clean all
```

Looks like we could also add a `yum clean all` here, similar to https://github.com/rapidsai/docker/blob/branch-0.20/templates/rapidsai-core/partials/patch.dockerfile.j2#L9